blocks/contact-form/contact-form.js

```diff
@@ -20,30 +32,39 @@ export default function decorate(block) {
       div.classList.add('contact-form__field');
     }
 
+    // Selectors
+    const inputTypeColumn = div.querySelector('div:last-of-type');
+
     // Add inputs with fitting "type" attributes to every field except for the title
     if (!fieldName.includes('title')) {
       // Create input element
       const input = document.createElement('input');
 
-      // Check for field name and add a fitting input "type" attribute
-      if (fieldName.includes('name')) {
-        input.setAttribute('type', 'text');
-      }
-      if (fieldName.includes('email')) {
-        input.setAttribute('type', 'email');
-      }
-      if (fieldName.includes('terms of service')) {
-        input.setAttribute('type', 'checkbox');
-      }
-      if (fieldName.includes('date')) {
-        input.setAttribute('type', 'date');
-      }
+      // Set the input type
+      input.setAttribute('type', inputTypeColumn.textContent);
```

Comment:
What if the content author accidentally adds "txet" as type? Or "select", because they think it's also supported?
Make sure to handle these cases somehow (at least with a console message)